src/utils.js

```diff
@@ -41,3 +43,26 @@ export const removeEmptyKeys = (obj) => {
 export function sleep(ms) {
   return new Promise((resolve) => setTimeout(resolve, ms));
 }
+
+export const updateFlagImage = (barcode, src) => {
```

Comment:
just wondering here (no clear idea), but are these two functions going to be used `updateFlagImage` and `removeFlagImage` in another place besides the questions? mmm think is better to have things as close to the place they are used and feel that having it here in a generic utils gets things a bit out of context. Please ignore this comment if you feel it is not very relevant

Reply:
I did that so that the API calls are separate. Should I just include it in the `removeFlagImage` file?